File: SuperToolTip_Disablecontrol_C#/Form1.cs
using Syncfusion.Windows.Forms;
using Syncfusion.Windows.Forms.Tools;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Diagnostics;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
namespace Supertooltip
{
    public partial class Form1 : MetroForm
    {
        private SuperToolTip superToolTip1;
        public Form1()
        {
           InitializeComponent();
           superToolTip1 = new SuperToolTip();

        }
        bool IsShown = false;
        private void Form1_MouseMove(object sender, MouseEventArgs e)
        {
            Control ctrl = this.GetChildAtPoint(e.Location);
            if (ctrl != null)
            {
                if (ctrl == this.buttonAdv1 && !IsShown)
                {
                    //Initializes the ToolTipInfo
                    ToolTipInfo tooltip = new ToolTipInfo();
                    tooltip.Body.Text = "SuperToolTip";
                    //Sets the SuperToolTip
                    this.superToolTip1.SetToolTip(this.buttonAdv1, tooltip);
                    System.Drawing.Point pt1 = new System.Drawing.Point(ctrl.Location.X + 20, ctrl.Location.Y + 30);
                    //Shows the SuperToolTip in the specified location
                    this.superToolTip1.Show(tooltip, this.PointToScreen(pt1), 1000);
                    IsShown = true;
                }
            }
            else
            {
                this.superToolTip1.Hide();
                IsShown = false;
            }
        }
    }
}

File: SuperToolTip_Disablecontrol_VB/Form1.vb
Imports Syncfusion.Windows.Forms
Imports Syncfusion.Windows.Forms.Tools

Public Class Form1
    Inherits MetroForm


    Private superToolTip1 As SuperToolTip
    Public Sub New()
        InitializeComponent()
        superToolTip1 = New SuperToolTip()

    End Sub
    Private IsShown As Boolean = False
    Private Sub Form1_MouseMove(ByVal sender As Object, ByVal e As MouseEventArgs) Handles MyBase.MouseMove
        Dim ctrl As Control = Me.GetChildAtPoint(e.Location)
        If ctrl IsNot Nothing Then
            If ctrl Is Me.buttonAdv1 AndAlso (Not IsShown) Then
                'Initializes the ToolTipInfo
                Dim tooltip As New ToolTipInfo()
                tooltip.Body.Text = "SuperToolTip"
                'Sets the SuperToolTip
                Me.superToolTip1.SetToolTip(Me.buttonAdv1, tooltip)
                Dim pt1 As New System.Drawing.Point(ctrl.Location.X + 20, ctrl.Location.Y + 30)
                'Shows the SuperToolTip in the specified location
                Me.superToolTip1.Show(tooltip, Me.PointToScreen(pt1), 1000)
                IsShown = True
            End If
        Else
            Me.superToolTip1.Hide()
            IsShown = False
        End If
    End Sub
End Class
